File: app/src/main/java/com/gmail/fattazzo/formula1world/domain/F1LapTime.java
package com.gmail.fattazzo.formula1world.domain;

/**
 * @author fattazzo
 *         <p/>
 *         date: 11/07/17
 */
public class F1LapTime {

    public F1Race race;

    public F1Driver driver;

    public int lap;

    public int position;

    public String time;

    public int milliseconds;

    @Override
    public String toString() {
        return "LapTime{" +
                "race=" + race +
                ", driver=" + driver +
                ", lap=" + lap +
                ", position=" + position +
                ", time='" + time + '\'' +
                ", milliseconds=" + milliseconds +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/domain/F1LapTime.kt
package com.gmail.fattazzo.formula1world.domain

/**
 * @author fattazzo
 *
 *
 * date: 11/07/17
 */
class F1LapTime {

    var race: F1Race? = null

    var driver: F1Driver? = null

    var lap: Int = 0

    var position: Int = 0

    var time: String? = null

    var milliseconds: Int = 0

    override fun toString(): String {
        return "LapTime{" +
                "race=" + race +
                ", driver=" + driver +
                ", lap=" + lap +
                ", position=" + position +
                ", time='" + time + '\'' +
                ", milliseconds=" + milliseconds +
                '}'
    }
}
